consensus/rewards.md

```diff
@@ -0,0 +1,35 @@
+# Miners' rewards
+
+Aeternity is Proof of Work blockchain. This is a description of rewards paid to miners.
+
+## Introduction
+
+Miners mine key blocks and receive rewards composed out of two ingredients:
+1. Coinbase-like reward for mining a key block
+2. Fees and gas paid by users for transactions and contract calls
+
+There is also special case altering rewarding scheme. It crypto economic incentive designed to lower risk of fraud - and it is called Proof of Fraud.
+
+Miners rewards are also affected by protocol protected mechanism of reward split.
+
+### Reward for mining a key block
+
+Miners include beneficiary address in a key block. The reward is sent to it with protocol protected delay.
+The size of the reward is computed based on inflation curve
+
+### Fees and gas
+
+Users need to pay for execution of their actions. According to Bitcoin-NG protocol a miner gets 60% of previous generation fees and 40% of the next one.
```

Comment:
This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L39-L46).